src/app/contexts/RequestContext/index.jsx

```diff
@@ -68,6 +68,7 @@ RequestContextProvider.propTypes = {
     'MAP',
     'FIX',
     'STY',
+    'PGL',
```

Comment:
Unrelated to this really but is there anywhere where all of these 3 letter codes are explained? Might be worth a link in the README?

Reply:
@tochwill good question, there is an extensive list on BBC confluence under `CPS+v6+Asset+Types`. The ones we are likely to come across are: 

| Type | Description | Example |
| ----- | ----------- | --------- |
| STY  | Basic story asset | https://www.bbc.co.uk/news/uk-england-hampshire-50232692 |
| MAP | Media asset page | https://www.bbc.co.uk/news/av/uk-39663004/how-do-i-register-to-vote |
| PGL  | Photo Gallery | https://www.bbc.co.uk/news/in-pictures-49918450 |
| CSP | Correspondent Piece | https://www.bbc.co.uk/news/technology-50149719 |
| LIV   | BBC Live Asset | https://www.bbc.co.uk/news/live/uk-politics-50212212 |
| IDX   | Editorial curated index asset | https://www.bbc.co.uk/news |
| FIX    | Feature index asset | https://www.bbc.co.uk/news/uk-40294993 |
| COP | Correspondent Page - similar to an index | https://www.bbc.co.uk/news/correspondents/davelee |

These are all entire pages where as some of the other types can be an item asset rather than an entire page.

Reply:
Thank you! This page is really helpful

Reply:
I think `COP` is `CSP`?